### PR TITLE
NOISSUE - Update Documentation on Emitted Events

### PR DESCRIPTION
**Pull request by rodneyosodo**
Signed-off-by: Rodney Osodo <socials@rodneyosodo.com>

### What does this do?
Update documentation of emitted events by mainflux
Moves events to separate page

### Which issue(s) does this PR fix/relate to?
No issue

### List any changes that modify/break current functionality
None

### Have you included tests for your changes?
No

### Did you document any new/modified functionality?
Yes

### Notes
To be merged after https://github.com/mainflux/mainflux/pull/1900

### CODE REVIEW

File: docs/events.md [hunk 1]
```diff
@@ -0,0 +1,1414 @@
...
+```
+
+### User issue token event
+
+Whenever user token is issued, `users` service will generate new `issue_token` event. This event will have the following format:
+
+```redis
+1) "1693307171987-0"
+2) 1) "operation"
+    2) "user.issue_token"
```
Comment (drasko) [inline docs/events.md:191]: Is this copy-pasted from somewhere? Why is it not aligned?

Comment (rodneyosodo) [reply docs/events.md:191]: Yes. From redis cli

**Review by dborovcanin — APPROVED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Comment by drasko** [comment]
@rodneyosodo @dborovcanin we have no info here regarding Nats JetStream event sourcing, only Redis. Is this OK?

**Comment by dborovcanin** [comment]
> @rodneyosodo @dborovcanin we have no info here regarding Nats JetStream event sourcing, only Redis. Is this OK?

Since NATS is still not merged, it sounds OK. Except if @rodneyosodo planned on adding it here, but forgot?

**Comment by rodneyosodo** [comment]
Documentation about NATS Jetstream Will be added on another PR

**Pull request merged**